modules/sdk-coin-atom/src/lib/utils.ts

```diff
@@ -225,6 +219,31 @@ export class Utils implements BaseUtils {
     const signedTxBytes = TxRaw.encode(signedTxRaw).finish();
     return toBase64(signedTxBytes);
   }
+
+  isSignedRawTx(rawTransaction: string): boolean {
+    // if (!this.isValidRawTransaction(rawTransaction)) {
+    //   return false;
+    // }
+    const decodedTx = this.getDecodedTxFromRawBase64(rawTransaction);
+    if (decodedTx.authInfo.signerInfos.length > 0) {
+      return true;
+    }
+    return false;
+  }
+
+  getSignerInfoFromRawSignedTx(rawTransaction: string) {
+    if (!this.isSignedRawTx(rawTransaction)) {
```

Comment:
This method gets called within `fromRawTransaction` in the transaction class. Before this method is called, we already check if the transaction is indeed signed or not (Line 123 in transaction class), so we might not need line 235 (unless this method can be called from elsewhere where we need to check if the transaction is signed)

Reply:
Let's keep this function for now, as we may call it in another class in the future!

Reply:
Good call - I was trying to explicitly show in this version that currently we are only supporting signed tx data, and are working on unsigned with the HSM team